style.css

```diff
@@ -370,7 +392,7 @@ hr {
   padding-right: 2rem;
 }
 
-.imgd1 {
+img.imgd1 {
   display: none;
 }
```

Comment:
- [ ] Kindly the images for each project don't appear as shown on the pic, kindly remove this `display: none;` to make them appear.

![image](https://user-images.githubusercontent.com/35535130/163176013-6cd76b0e-48a5-47ee-9a41-ccaa9d19e3ba.png)